### PR TITLE
WIP to master branch

### PR DESCRIPTION
**Pull request by orgicus**
Adding progress up to date on Runway Processing Library:

- basic OSC support
- HTTP support with the ability to change host and port
- many HTTP examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,94 @@
...
-  * Enter your GitHub credentials in the "Authentication" group, and click "Next >".
-  * Select the `master` branch on the next screen, and click "Next >".
-  * The default settings on the "Local Configuration" screen should work fine, click "Next >".
-  * Make sure "Import existing projects" is selected, and click "Next >".
-  * Eclipse should find and select the `processing-library-template` automatically, click "Finish".
-1. Rename your Eclipse project.
-  * In the Package Explorer, right-click (ctrl-click) on the folder icon of the `processing-library-template` project, and select Refactor → Rename... from the menu that pops up. 
-  * Give the project the name of your Library, and click "OK".
+<p align="center">
+  <img src="./assets/cover_runway_processing_github.jpg">
```
Comment (cvalenzuela) [inline README.md]: We can remove this cover for now, we are working on a new cover iteration for libraries.

File: README.md [hunk 2]
```diff
@@ -1,100 +1,94 @@
...
+</p>
+
+# Runway Processing library source
+
+A library to easily use [Runway](https://runwayml.com) with [Processing](https://processing.org/).
```
Comment (cvalenzuela) [inline README.md]:
```suggestion
A library to easily use [RunwayML](https://runwayml.com) with [Processing](https://processing.org/).
```

File: README.md [hunk 3]
```diff
@@ -1,100 +1,94 @@
...
+
+## Examples
+
+- AdaIN-Style-Transfer ![preview](assets/examples/ada-in-style-transfer.jpg)
```
Comment (cvalenzuela) [inline README.md:50]: All this examples are super cool! Thanks for taking the time to make them

File: README.md [hunk 4]
```diff
@@ -1,100 +1,94 @@
...
+
+- StyleGAN: ![preview](assets/examples/style-gan.jpg)
+
+- VisualImportance
+
+- YOLACT
+  
 
-## What is the difference between JDK and JRE?
+## Dependencies
```
Comment (cvalenzuela) [inline README.md:109]: Perhaps a "Getting Started" section would make sense but I think it's better if we merge this PR first and then add a few resources for people to get started in another PR. Let's keep it like this for now.

File: README.md [hunk 5]
```diff
@@ -1,100 +1,94 @@
...
+This is still a work in progress. Contributions are welcomed!
 
-To change the JRE used to compile your Java project:
+## Credits
 
-1. Open the properties of your project from the menu Project → Properties. Select "Java Build Path" and in its submenu, click on the "Libraries" tab.
-1. A list of JARs and class folders in the build path will show up. In this list you can find the JRE System Library that is used to compile your code. Remove this JRE System library.
-1. Click "Add Library...". In the popup window, choose "JRE System Library" and press "Next".
-1. Select an alternate JRE from the pull-down menu or click and modify the "Installed JREs". Confirm with "Finish" and "OK". 
+Special thanks for mentoring and support from [Cris Valenzuela, Anastasis Germanidis](https://runwayml.com) and [Daniel Shiffman](https://github.com/shiffman)
```
Comment (cvalenzuela) [inline README.md:121]: ❤️

File: resources/build.properties [hunk 6]
```diff
@@ -59,13 +59,13 @@ ant.description=Processing Library Ant build file.
 # Give your Library a name. The name must not contain spaces or special 
 # characters.
 
-project.name=YourLibrary
+project.name=Runway
```
Comment (cvalenzuela) [inline resources/build.properties]:
```suggestion
project.name=RunwayML
```

**Review by cvalenzuela — APPROVED** [review]
Amazing work! this is great and it's gonna be super helpful for the community! Great contribution

**Comment by cvalenzuela** [comment]
YES! I think this is ready for a merge! Amazing work @orgicus

**Comment by shiffman** [comment]
Wooho!! Excited to work with it!

**Pull request merged**